test/index.js

```diff
@@ -33,25 +33,52 @@ function Just(x) {
   return {constructor: MaybeTypeRep, isNothing: false, isJust: true, value: x};
 }
 
+function mock($$type) {
+  return {constructor: {'@@type': $$type}};
+}
+
+function parsed(namespace, name, version) {
+  return {namespace: namespace, name: name, version: version};
+}
```

Comment:
Would be nicer, but it somewhat defeats the point of the function, as to causes many lines to exceed 79 characters.